src/AMQP/CQRS/ICommand.ts

```diff
@@ -1,13 +1,13 @@
 
-export interface ICommandPayload {
-	type: string;
+export interface ICommandPayload<T extends string> {
+	type: T;
 }
 
-interface ICommand<TPayload extends ICommandPayload = ICommandPayload> {
```

Comment:
Tady jak jsem někam psal, tohle není syntax error. V TS 2.3 to je defaultní hodnota pro typový argument.